### DISCUSSION TITLE
Correspondent, document type selection

### DISCUSSION THREAD

**Discussion by tural-ali** [icereed/paperless-gpt#28] [Ideas]
Hi,
First would like to thank you for this fantastic software. 
The only thing that is really missing for me  - is the lack of support for fields like correspondent and document type. 
Also would like to have multiple language support. 
Are you going to cover them as well, or better me to do? :)

**Comment by icereed** [comment]
Hi @tural-ali,

thank you for your feedback ☺️
Document types should be similar to how tags are implemented. However, correspondent might be more complicated. If there is no name of a person in a document the LLM would need to guess based on the knowledge of the persons. That in turn means we would need to describe people to the LLM inside the prompt so it can guess correctly. 

Example: Bob likes botanics, Anne likes guitars. Then a specific guitar warranty document should be linked to Anne, where as a receipt for 10 flowers should be linked to Bob. Without this knowledge the LLM might decide based on stereotypes.


Other topic: I agree on the multi language capability. However, I’m not sure how I need to prioritize this compared to other feature requests. Maybe we get more feedback on this.
